ADAL/src/ADEnrollmentGateway.h

```diff
@@ -29,6 +29,8 @@
 + (NSString *)allEnrollmentIds;
```

Comment:
I am thinking that, should we rename it `allEnrollmentIdsJSON`? indicating it is returning a JSON blob. What do you think? :)